sdk/eventhub/eventhubs-checkpoint-blob/package.json

```diff
@@ -0,0 +1,125 @@
+  "sideEffects": false,
+  "keywords": [
+    "azure",
+    "cloud",
+    "event hubs",
+    "events"
+  ],
+  "bugs": {
+    "url": "https://github.com/azure/azure-sdk-for-js/issues"
+  },
+  "main": "./dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "types": "./typings/eventhubs-checkpoint-blob.d.ts",
+  "browser": {
+    "./dist/index.js": "./browser/index.js"
+  },
+  "engine": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "src/",
+    "typings/eventhubs-checkpoint-blob.d.ts",
+    "tsconfig.json"
+  ],
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:samples": "cd samples && tsc -p .",
+    "build:test:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test": "npm run build:test:node",
+    "build": "tsc -p . && rollup -c 2>&1 && npm run extract-api",
+    "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm test-dist typings *.tgz *.log",
+    "coverage": "nyc --reporter=lcov --exclude-after-remap=false mocha -t 120000 test-dist/index.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "mocha -t 1200000 test-dist/index.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint -c ../../.eslintrc.json src test samples --ext .ts --fix",
+    "lint": "eslint -c ../../.eslintrc.json src test samples --ext .ts -f html -o event-hubs-lintReport.html || exit 0",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "echo skipped",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "dependencies": {
+    "@azure/event-hubs": "5.0.0-preview.3",
+    "@azure/storage-blob": "12.0.0-preview.2",
+    "@azure/core-http": "1.0.0-preview.3",
```

Comment:
It doesn't look like you actually depend on `@azure/core-http` anywhere, so this could be removed.